src/k8s/go.mod

```diff
@@ -1,31 +1,31 @@
 module github.com/canonical/k8s
 
-go 1.22.6
+go 1.22.7
 
 require (
 	dario.cat/mergo v1.0.0
-	github.com/canonical/go-dqlite v1.22.0
+	github.com/canonical/go-dqlite/v3 v3.0.0
 	github.com/canonical/k8s-snap-api v1.0.13
-	github.com/canonical/lxd v0.0.0-20240822122218-e7b2a7a83230
-	github.com/canonical/microcluster/v3 v3.0.0-20240827143335-f7a4d3984970
+	github.com/canonical/lxd v0.0.0-20241106103908-9ac2433510c8
+	github.com/canonical/microcluster/v3 v3.0.0-20241106092713-7f54bbd37f6f
```

Comment:
For our direct dependency on `go-dqlite` we'll requre/use `v2` just like `k8s-dqlite`. 

For microcluster sadly seems like we can't get away with `replace github.com/canonical/go-dqlite v1.22.0 => github.com/canonical/go-dqlite/v2 v2.0.0` since there is a module path change. This means we'd have to fork microcluster, patch to use v2 paths on top of our pinned commit which is not feasible. Instead we'll stick with `v3` which will indirectly require go-dqlite `v3`. (Well unless we have breaking changes there)

`go mod tidy` should sort things out.

Reply:
Ignore the comments, was not aware of the build issues.